pytket/extensions/qiskit/backends/aer.py

```diff
@@ -440,6 +455,9 @@ def __init__(
          https://qiskit.org/documentation/stubs/qiskit.providers.aer.AerSimulator.html
          for available values. Defaults to "automatic".
         :type simulation_method: str
+        :param crosstalk_params: Apply crosstalk noise simulation to the circuits before
```

Comment:
I would updated the `crosstalk_params` description to explain that any other noise model will be overwritten.